Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/IncompleteProjectsGetDataQueryTests.cs

```diff
@@ -47,16 +47,16 @@ public async Task ProjectExists___GetProject()
 			);
 		}
 
-		private (ProjectDetails, ProjectState) CreateTestProject(string? localAuthority = null, string? region = null)
+		private (ProjectDetails, Project) CreateTestProject(string? localAuthority = null, string? region = null)
 		{
 			var projectDetails = _fixture.Build<ProjectDetails>()
 				.With(p => p.LocalAuthority, localAuthority)
 				.With(p => p.Region, region)
 				.Create();
 			var newProject = new Project(0, projectDetails);
-			var mappedProject = ProjectState.MapFromDomain(newProject);
+			//var mappedProject = ProjectState.MapFromDomain(newProject);
```

Comment:
If it doesn't need to be mapped lets remove comment

Reply:
removed